doc/services/device_mgmt/ec_host_cmd.rst

```diff
@@ -120,10 +120,10 @@ Logging
 The host command has an embedded logging system of the ongoing communication. The are a few logging
 levels:
 
-* `LOG_INF` is used to log a command id of a new command and not success responses. Repeats of the
+* ``LOG_INF`` is used to log a command id of a new command and not success responses. Repeats of the
   same command are not logged
-* `LOG_DBG` logs every command, even repeats
-* `LOG_DBG` + :kconfig:option:`CONFIG_EC_HOST_CMD_LOG_DBG_BUFFERS` logs every command and responses
+* ``LOG_DBG`` logs every command, even repeats
+* ``LOG_DBG`` + :kconfig:option:`CONFIG_EC_HOST_CMD_LOG_DBG_BUFFERS` logs every command and responses
```

Comment:
There are functions/macros; should they use the macro or func refs?

Reply:
absolutely -- an oversight :)